### PR TITLE
add a link to the EPC blog post

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Eduardo Arango <eduardo@sylabs.io>

**Description of the Pull Request (PR):**

*EPC* acronym for _Enterprise Performance Computing_ is not explained on the `README.md`, so a link to our definition of it is helpful for new users 

**This fixes or addresses the following GitHub issues:**

- Fixes #

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](../CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](../CHANGELOG.md) if necessary according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](../CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](../CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

**Review by WestleyK — APPROVED** [review]
Looks good to me :+1:

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 - [Documentation](https://www.sylabs.io/docs/)
 - [Citation](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0177459)
 
-Singularity is an open source container platform designed to be simple, fast, and secure. Singularity is optimized for EPC and HPC workloads, allowing untrusted users to run untrusted containers in a trusted way.
+Singularity is an open source container platform designed to be simple, fast, and secure. Singularity is optimized for [EPC](https://www.sylabs.io/2018/09/singularity-is-enterprise-performance-computing/) and HPC workloads, allowing untrusted users to run untrusted containers in a trusted way.
 
 Check out [who is using Singularity](https://www.sylabs.io/singularity/whos-using-singularity/) and some [use cases of Singularity](https://www.sylabs.io/category/how-tos/) on our website.
```
Comment (WestleyK) [inline README.md:15]: Should this link to our [sylabs/examples](https://github.com/sylabs/examples)?

Comment (ArangoGutierrez) [reply README.md:15]: that's `who is using` singularity, not `how to use it`

Comment (WestleyK) [reply README.md:15]: This link:

```
[use cases of Singularity](https://www.sylabs.io/category/how-tos/)
```

Should this link to our example repo?

Comment (ArangoGutierrez) [reply README.md:15]: umm you are right. well the link is a web version of the examples repo ( on lab note style)

**Review by jchavezlavalle — APPROVED** [review]

**Review by GodloveD — APPROVED** [review]
lgtm.  merge when checks complete

**Review by GwendolynK-zz — APPROVED** [review]
Looks good to me!

**Pull request merged**